packages/ui/src/components/ProjectCard/ProjectCard.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { SimpleGrid } from '@chakra-ui/react';
+import { Meta, Story } from '@storybook/react';
+
+import { ProjectCard, ProjectCardType } from './ProjectCard';
+
+const title = 'Zaawansowany React i Node';
+const date = new Date();
+
+const status = { done: 'done', idle: 'idle', doing: 'doing', review: 'review' };
+const meta: Meta = {
+  title: 'Components / ProjectCard',
+  component: ProjectCard,
+  argTypes: {
+    status: {
+      options: Object.keys(status),
+      mapping: status,
+      control: {
+        type: 'select',
+      },
+    },
+  },
+};
+
+export default meta;
+
+const Template: Story<ProjectCardType> = (args) => (
+  <SimpleGrid columns={4} spacing={5} h="444px">
+    <ProjectCard {...args} />
+  </SimpleGrid>
+);
+
+export const Playground = Template.bind({});
+
+Playground.args = {
+  status: 'idle',
+  image: 'test',
+  title: 'Podstawy web developmentu',
+  url: '/test',
+  date,
+  points: 100,
+  pointsMax: 200,
+};
+
+export const Variants = () => (
+  <SimpleGrid columns={4} spacing={3} h="444px" w="320">
+    <ProjectCard status="idle" image="test" title={title} url="test" date={date} />
+    <ProjectCard status="doing" image="test" title={title} url="test" date={date} />
+    <ProjectCard status="review" image="test" title={title} url="test" date={date} />
+    <ProjectCard status="done" image="test" title={title} url="test" points={100} pointsMax={200} />
+  </SimpleGrid>
```

Comment:
🏕️Same as said in previous comment about enums for status (or constants)
The other thing is you declared the object with values for status props and you still used plain strings in here, I think we should refactor that :D